### PR TITLE
Refined and more proper WebView element types

### PR DESCRIPTION
**Pull request by niikoo**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://developer.chrome.com/apps/tags/webview
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by niikoo** [comment]
Seems like the typescript-bot didn't tag the owners. @matthewkimber @otiai10 @couven92 @RReverser @sreimer15 @niikoo @AdamLay @pine613 @mzsm @pyle could any of you please review this PR?

**Comment by pyle** [comment]
Taking a look.

File: types/chrome-apps/index.d.ts [hunk 1]
```diff
@@ -1526,23 +2396,33 @@ declare namespace chrome {
...
-     * Permissions:  "gcm"
+     * Permissions:  'gcm'
      */
     namespace gcm {
+        interface IGCMData {
+            'collapse_key'?: never;
+            'goog'?: never;
+            'goog.'?: never;
+            'google'?: never;
+            'GOOGLE'?: never;
```
Comment (pyle) [inline types/chrome-apps/index.d.ts]: Why change this to wrap the property names in string?

File: types/chrome-apps/test/index.ts [hunk 2]
```diff
@@ -324,5 +324,40 @@ function testSystemNetwork() {
...
-let element: webview.HTMLWebViewElement;
+const gcmMessage = <chrome.gcm.OutgoingMessage>{};
+gcmMessage.data = {
+    /*goog: 'any', should not be allowed, and it is not :) */
+    test: true
+};
+
+let wve: chrome.webview.HTMLWebViewElement = (<any>document.getElementById('webview'));
+wve.name = 'test';
+wve.src = 'http://github.com/niikoo'
```
Comment (pyle) [inline types/chrome-apps/test/index.ts]: Might be better to add the repo site here instead incase your username changes etc
http://github.com/DefinitelyTyped

**Review by pyle — CHANGES_REQUESTED** [review]

File: types/chrome-apps/test/index.ts (hunk 2 above)
Comment (niikoo) [reply types/chrome-apps/test/index.ts]: Thank you for your feedback.
It's now changed and committed.

**Review by pyle — APPROVED** [review]

**Pull request merged**